test/Libraries/PackageManagerTests/PackageManagerTests.csproj

```diff
@@ -1,5 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
-  <ImportGroup Label="PropertySheets">
+    <PropertyGroup>
+      <UILib>true</UILib>
```

Comment:
@sm6srw @JimmySanchezGlobant if you guys have not touched base I think this will solve Jimmy's issue as well.

Reply:
@sm6srw do we need to now remove these tests from the core test script?

Reply:
@mjkkirschner I tried before but unfortunately it didn't work.

Reply:
This works for me but we are running these tests in the Linux build so this not the final solution.  The question is where to move this test?  A new test project just for one test? I am kind of undecided right now.

Reply:
@mjkkirschner Ok, now understand your comment. Yes, stop running these tests in the Linux build is an option.

Reply:
We may want to split PackageManager tests into Core and UI so that it can also run on linux at some point in the future. For now this is ok with me